.github/scripts/directory_check.py

```diff
@@ -62,7 +65,7 @@ def get_md_files(dir_config, path):
             if md_name.startswith(('http://', 'https://')):
                 continue
             elif md_name == './':
-                md_list.append(f'{docs_path}/{path}/README.md')
+                md_list.append(f'{docs_path}/{path}/index.md')
```

Comment:
it seems that you have to change the name of 'README.md' into 'index.md' to pass ci.